sdk/nodejs/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "@pulumi/xyz",
+    "version": "${VERSION}",
```

Comment:
I was surprised that we inject `$"{VERSION}"` here.  That forces consumers to put some non-trivial tooling around the generation of this package.  

I suppose that we need a decent amount of tooling to support build and publish anyway.   How can we make that "easy"?

Reply:
Ah, interesting. I would have expected this to come from the schema as well.

Reply:
Opened https://github.com/pulumi/pulumi-provider-boilerplate/issues/3 on the build/publish machinery we'll need to add to make this work.  

Ideally we would also just simplify the Node.js code generation here - but that may be a task for another day.